comment/api/urls.py

```diff
@@ -3,16 +3,18 @@
 
 from comment.api import views
 
+app_name = 'comment-api'
```

Comment:
since the `app_name` wasn't present earlier in this module, the subsequent url names in this module were literally unusable. so this shouldn't be an issue as far as i can think.

Reply:
I think the same, just wanted to make sure.